dae-alright/src/components/Button/Button.component.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './Button.style.scss';
+
+const Button = ({btnDetails,bgColor, color, icons, width, height, className}) => (
+
+    <button className={`${className}`} style={{
+        backgroundColor:bgColor,
```

Comment:
You can do `style={{...style, padding: '3px'}}`

Reply:
Done, Commit link: aae0f32ce7a3bebbdea9aa241f946689d2bf21c8